smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/RuntimeClientPlugin.java

```diff
@@ -389,6 +434,35 @@ public Builder resolveFunction(Symbol resolveFunction) {
             return resolveFunction(SymbolReference.builder().symbol(resolveFunction).build());
         }
 
+        /**
+         * Sets the symbol that is invoked in order to convert the
+         * input symbol type to a resolved symbol type.
+         *
+         * <p>If this is set, then both {@link #resolvedConfig} and
+         * {@link #inputConfig} must also be set.
+         *
+         * @param resolveFunction Function used to convert input to resolved.
+         * @param additionalParameters Additional parameters to be generated as resolve function input.
+         * @return Returns the builder.
+         * @see #getResolveFunction()
+         */
+        public Builder resolveFunction(Symbol resolveFunction, String... additionalParameters) {
+            return resolveFunction(SymbolReference.builder().symbol(resolveFunction).build(), additionalParameters);
+        }
+
+        /**
+         * Add additional positional input parameters to resolve function
```

Comment:
This method is setting the parameters, not adding to the list. The name (remove `add`) and documentation should be updated to clarify this.

Is there use for singular `add` and `remove`, `clear`, or an option that takes a list instead of a variadic?